keybase/platform_windows.go

```diff
@@ -10,10 +10,22 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kardianos/osext"
 	"github.com/keybase/go-updater"
 	"github.com/keybase/go-updater/command"
 )
 
+// execPath returns the app bundle path where this executable is located
+func (c config) execPath() string {
```

Comment:
The intent is for the updater to always find prompter.hta in a subdirectory named "prompter" under the current location from which upd.exe was run. Open to other suggestions of where it lives and how it gets found.